README.md

```diff
@@ -1,32 +1,39 @@
-# Description
-This is a dev environment based on Docker, as an alternative to the homestead virtualization which already exists. Docker works best on Linux, we as developers are mainly using Ubuntu to test it. With a Linux operating system installation of the system is really simple, if you do not have Linux you can look into setting up a Virtual Machine yourself or trying out our Vagrant option, which is at the time of writing not 100% stable.
+# AEGEE-Europe's Online Membership System
+## OMS-docker
 
-# Installing
+## Description
+The provisional repository for the [Online Member System (OMS)](https://github.com/AEGEE/oms-docker), an open-source project of student association [AEGEE-Europe](http://aegee.org/).
 
-Linux:
+It makes use of docker, and docker-compose.
+
+[Read more about it.](https://oms-project.atlassian.net/wiki/spaces/GENERAL/overview)
+
+## Installing
+
+[Install docker and docker-compose](https://docs.docker.com/compose/install/)
+
+Install OMS:
 ```
 git clone --recursive https://github.com/AEGEE/oms-docker.git
-cd oms-docker/docker-dev
+cd oms-docker/docker
 sudo docker-compose up
 ```
 
-NOTE: Because Traefik works reading the URL of the request, it is necessary to add to your `/etc/hosts` file a name for your address (127.0.0.1 if bare Docker, 192.168.192.168 if Vagrant) - Traefik is configured under the name **appserver**.
-
-Open you browser on `http://appserver` and login with the default credentials. Navigate to the modules tab and activate all the microservices
-
+*This is assuming you are running a Linux installation, for other operating systems virtualization is recommended.*
```

Comment:
The more detailed installation instructions contain a link to an article about it:
https://oms-project.atlassian.net/wiki/spaces/GENERAL/pages/17235970/Installation

If you still want to do vagrant stuff feel free, but to be honest I felt like this was the simpler solution: giving the User more freedom in terms of how to get Ubuntu running, be at native, VirtualBox, Vagrant, or any other way. Besides, setting up VirtualBox is not really hard and should be well within the capabilities of someone supposed to work on the project.